react/next/utils/throwError.ts

```diff
@@ -0,0 +1,19 @@
+import { SliderProps } from '../typings'
+
+const throwError = (props: SliderProps): any => {
+  const { ssr, responsive } = props
+
+  if (!responsive) {
+    throw new Error(
+      ssr
+        ? '⚠️ SSR mode need to be used in conjunction with responsive prop'
+        : '️️️️⚠️ The responsive prop is when on SSR mode'
```

Comment:
I don't understand what you want to say here

Reply:
I've merged `ssr` and `responsive` props because they only make sense when used together.